OEPS/OEP-9.mediawiki

```diff
@@ -0,0 +1,207 @@
+<pre>
+  OEP: 9
+  Title: Ontology Wallet Standard
+  Author: Wyatt Mufson <wyatt@towerbuilders.org>
+  Type: Standard
+  Status: Draft
+  Created: 2019-07-07
+</pre>
+
+==Abstract==
+
+The OEP-9 Proposal is a standard for managing wallet files across all Ontology applications.
+
+==Motivation==
+
+There is currently no OEP defining a wallet standard for Ontology.
+Different SDKs list their own wallet file specification.
+While most are the same, there is not one standard that has been determined.
+An established wallet file standard will improve interoperability between applications.
+
+==Specification==
+
+For simplicity I have taken the wallet file specification outlined in the <code>ontology-java-sdk</code>.
+
+===Wallet===
+
+A wallet file in JSON format has the following basic structure:
+
+<pre>
+{
+  "name": "MyWallet",
+  "version": "1.0",
+  "scrypt": {},
+  "defaultOntid": "did:ont:TJNxJe2Ty8eSCjCDxDPwiS78NRAn8XPTFL",
+  "defaultAccountAddress": "TJNxJe2Ty8eSCjCDxDPwiS78NRAn8XPTFL",
```

Comment:
Although, ONT-ID can be same with account address.  
Suggest making them different in OEP, to avoid some misleading :)

Reply:
Agreed - made the switch